src/Altinn.Notifications/Altinn.Notifications.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Common.AccessToken" Version="3.0.3" />
+    <PackageReference Include="Altinn.Common.AccessToken" Version="4.1.0" />
```

Comment:
Now that I understand a bit better what it is that is breaking Authorization I believe this should work. Notification doesn't have ABAC based Authorization as far as I can remember.

For application with ABAC based Authorization I hope updating PEP to 1.4.0-alpha3 will fix the issues we've been having. We can try this in Events if you want.

Reply:
will test in events.